components/x-privacy-manager/src/privacy-manager.jsx

```diff
@@ -1,207 +1,93 @@
-					credentials: 'include'
-				})
-
-				// On response call any externally defined handlers following Node's convention:
-				// 1. Either an error object or `null` as the first argument
-				// 2. An object containing `consent` and `payload` as the second
-				// Allows callbacks to decide how to handle a failure scenario
-
-				if (res.ok === false) {
-					throw new Error(res.statusText || String(res.status))
-				}
-
-				for (const fn of onConsentSavedCallbacks) {
-					fn(null, { consent, payload })
-				}
-
-				return { _response: { ok: true } }
-			} catch (err) {
-				for (const fn of onConsentSavedCallbacks) {
-					fn(err, { consent, payload })
-				}
-
-				return { _response: { ok: false } }
-			}
-		}
-	}
-}))
-
-/**
- * @param {boolean} isLoading
- * @param {_Response} response
- * @param {string} referrer
- */
-function renderMessage(isLoading, response, referrer) {
-	if (isLoading) return <LoadingMessage />
-	if (response) return <ResponseMessage success={response.ok} referrer={referrer} />
-	return null
-}
-
-/**
- * Display a warning to users
- * @param {string | undefined} userId
- */
-function renderLoggedOutWarning(userId) {
-	if (userId && userId.length > 0) return null
 
-	return (
-		<p className={`${s.consent__copy} ${s['consent__copy--cta']}`}>
-			Please sign into your account before submitting your preferences to ensure these changes are applied
-			across all of your devices
-		</p>
-	)
+const defaultButtonText = {
+	allow: { label: 'Allow', text: 'See personalised adverts' },
+	block: { label: 'Block', text: 'Opt out of personalised adverts' },
+	submit: { label: 'Save' }
 }
 
 /**
- * @param {BasePrivacyManagerProps} Props
+ * @param {import('../typings/x-privacy-manager').BasePrivacyManagerProps} Props
  */
-
 export function BasePrivacyManager({
 	userId,
-	consent = true,
-	consentProxyEndpoints,
-	consentSource,
-	onConsentSavedCallbacks = [],
 	referrer,
+	legislationId,
+	cookiesOnly,
 	cookieDomain,
+	fow,
+	consent,
+	consentSource,
+	consentProxyApiHost,
+	onConsentSavedCallbacks = [],
+	buttonText = {},
+	loginUrl,
 	actions,
 	isLoading,
 	_response = undefined
 }) {
-	const trackingAction = consent ? 'allow' : 'block'
-	const btnTrackingId = `ccpa-advertising-consent-${trackingAction}`
+	// Shallowly merge supplied button labels with defaults
+	buttonText = { ...defaultButtonText, ...buttonText }
+
+	const consentProxyEndpoints = utils.getConsentProxyEndpoints({
+		userId,
+		consentProxyApiHost,
+		cookiesOnly,
+		cookieDomain
+	})
+	const consentApiUrl = consentProxyEndpoints.createOrUpdateRecord
+	const trackingKeys = utils.getTrackingKeys(legislationId)
+	const { sendConsent, onConsentChange } = actions
 
-	return (
-		<div className={s.consent}>
-			<h1 className={s.consent__title}>Do Not Sell My Personal Information</h1>
-			<div className={s.consent__copy}>
-				<p>
-					If you are a California resident, the California Consumer Privacy Act (CCPA) provides you with a
```

Comment:
why was this message removed?

Reply:
As part of the work to make the component region-agnostic: this messaging will be provided by the consuming app